packages/xod-project/src/internal/patchPathUtils.js

```diff
@@ -8,13 +8,31 @@ export const getExpandedVariadicPatchPath = R.curry(
     [originalPath, EXPANDED_PATCH_NAME_SUFFIX, arityLevel].join('')
 );
 
+// =============================================================================
+// RegExp parts
+// =============================================================================
+const alphanumeric = '[a-z0-9]+';
+const alphanumericWithHypens = `${alphanumeric}(-${alphanumeric})*`;
+const alphanumericWithHypensAndCommans = `${alphanumeric}(-${alphanumeric}|,${alphanumeric})*`;
+const variadicLevel = '-\\$\\d';
```

Comment:
I love the idea, but let’s remove this easter egg until we’ll make the resolution for the different variadic levels?

Reply:
It's just about PatchPath type, to validate flattened variadic nodes.
Early validation of patch paths did not work and so this rule appeared only now.